src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml.cs

```diff
@@ -77,7 +77,7 @@ private void ColumnHeader_Clicked(object sender, RoutedEventArgs e)
         public void SaveSettings(UserSettings settings)
         {
             var sortDescription = _projectList.Items.SortDescriptions.FirstOrDefault();
-            if (sortDescription != null)
+            if (sortDescription != default(SortDescription))
```

Comment:
:memo: This is a behavior change; please review to make sure this was the intent.